{"uuid": "2c58588c-fdff-4d8e-b889-b3193f14b790", "children": ["51f85472-9847-414c-a0c2-06c10cd04b44"], "befores": [{"name": "create_order", "status": "broken", "statusDetails": {"message": "requests.exceptions.ConnectionError: ('Connection aborted.', ConnectionResetError(10054, 'Удаленный хост принудительно разорвал существующее подключение', None, 10054, None))\n", "trace": "  File \"D:\\for_git\\projects\\Sprint_7\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"D:\\for_git\\projects\\Sprint_7\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"D:\\for_git\\projects\\Sprint_7\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\for_git\\projects\\Sprint_7\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1139, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\for_git\\projects\\Sprint_7\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 897, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\for_git\\projects\\Sprint_7\\conftest.py\", line 46, in create_order\n    response = requests.post(f'{BASE_URL}orders', json=order_data)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\for_git\\projects\\Sprint_7\\.venv\\Lib\\site-packages\\requests\\api.py\", line 115, in post\n    return request(\"post\", url, data=data, json=json, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\for_git\\projects\\Sprint_7\\.venv\\Lib\\site-packages\\requests\\api.py\", line 59, in request\n    return session.request(method=method, url=url, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\for_git\\projects\\Sprint_7\\.venv\\Lib\\site-packages\\requests\\sessions.py\", line 589, in request\n    resp = self.send(prep, **send_kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\for_git\\projects\\Sprint_7\\.venv\\Lib\\site-packages\\requests\\sessions.py\", line 703, in send\n    r = adapter.send(request, **kwargs)\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\for_git\\projects\\Sprint_7\\.venv\\Lib\\site-packages\\requests\\adapters.py\", line 682, in send\n    raise ConnectionError(err, request=request)\n"}, "start": 1727815749347, "stop": 1727815749635}], "start": 1727815749347, "stop": 1727815750019}